Source/App/EbAppConfig.c

```diff
@@ -502,7 +502,7 @@ void EbConfigCtor(EbConfig_t *configPtr)
     configPtr->baseLayerSwitchMode                  = 0;
     configPtr->predStructure                        = 2;
     configPtr->intraPeriod                          = -2;
-    configPtr->intraRefreshType                     = 1;
+    configPtr->intraRefreshType                     = 0;
```

Comment:
Hi Jing, please assign it with IDR_REFRESH (0). Thanks!

Reply:
It's not defined in this scope.